ant-cli/src/commands.rs

```diff
@@ -116,6 +117,15 @@ pub enum FileCmd {
         /// Experimental: Optionally specify the number of retries for the download.
         #[arg(short, long)]
         retries: Option<usize>,
+        /// Enable chunk caching to resume failed downloads.
+        /// Successfully downloaded chunks are cached to disk and reused on retry.
+        /// The cache is automatically keyed by the data address being downloaded.
+        #[arg(long)]
+        cache_chunks: bool,
```

Comment:
Since caching is the default, I think this option should be presented to the user as `--disable-cache`, which can then be inverted to `cache_chunks` for the code to deal with, since it's easier to read the positive case.

The documentation for the argument could be:
```
By default, chunks will be cached to enable resuming downloads.

Set this flag to disable the cache.
```